snappy/utils.go

```diff
@@ -37,7 +37,7 @@ import (
 // any already existing SNAP_* variables in os.Environment()
 func makeSnapHookEnv(part *SnapPart) (env []string) {
 	desc := struct {
-		AppName     string
+		SnapName    string
 		AppArch     string
 		AppPath     string
```

Comment:
Same, this is essentially a bit of untyped magic that is not type checked in the traditional sense. It gets expanded with Template in ``snap/snapenv/``